src/Control/HTTPRequest.php

```diff
@@ -406,8 +406,11 @@ public function getURL($includeGetVars = false)
      */
     public function isAjax()
     {
+        $contentType = $this->getHeader('Content-Type') ?: $this->getHeader('content-type');
```

Comment:
Shouldn't this be the `accept` header?

Better to use `$this->getAcceptMimetypes()`?